insights/parsers/httpd_conf.py

```diff
@@ -89,39 +77,39 @@
 from .. import Parser, parser, get_active_lines, LegacyItemAccess
 
 
+SUPPORTED_SECTION_BEG = ('<IfModule', '<VirtualHost')
+SUPPORTED_SECTION_END = ('</IfModule', '</VirtualHost')
+
+
 @parser('httpd.conf', filters=['IncludeOptional'])
 @parser('httpd.conf.d')
 class HttpdConf(LegacyItemAccess, Parser):
     """
     Get the key value pairs separated on the first space, ignoring leading
-    and trailing spaces.  The "<IfModule prefork.c>" and "<IfModule worker.c>"
-    sections are parsed into 'MPM_prefork' and 'MPM_worker' sub-dictionaries
-    respectively.
+    and trailing spaces.
 
     If the file is ``httpd.conf``, it also stores first half, before
     ``IncludeOptional conf.d/*.conf`` line, and the rest, to the ``first_half``
     and ``second_half`` attributes respectively.
 
     Attributes:
-        data (dict): Dictionary of parsed data in format of {option: value} or
```

Comment:
If all of the configuration is going to be stored in one parameter, that's fine, as long as it stores what was previously called `full_data`.  In that case, however, `get_setting_list` and `get_active_setting` should provide a clearer interface than referring to the setting list implicitly or referring to the `[-1]` index of that list.